src/styles/reset.css

```diff
@@ -0,0 +1,24 @@
+* {
+  box-sizing: border-box;
+}
+
+.page__content {
+  margin: 0;
+}
+
+h2 {
+  font-size: 1em;
+  margin: 0;
+  font-weight: normal;
+}
```

Comment:
split to reset and normalize
  margin: 0; - reset.css
  font-size: 1em; - normalize
  font-size: 1em; - normalize